tests/wdio.conf.js

```diff
@@ -1,3 +1,26 @@
+require('dotenv').load({silent: true});
```

Comment:
Had trouble getting Browserstack to work with a local running server. I've decided to stop that work as we can just as easily deploy to an ephemeral Heroku Dyno and point Browserstack to that instead.